tools/wait_for_assisted_service.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import argparse
+import waiting
+import requests
+
+import utils
+import deployment_options
+
+SERVICE = "assisted-service"
+TIMEOUT = 60 * 8
+SLEEP = 3
+
+def handle_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--target")
+
+    return deployment_options.load_deployment_options(parser)
+
+
+def main():
+    deploy_options = handle_arguments()
+
+    # TODO: delete once rename everything to assisted-installer
+    if deploy_options.target == "oc-ingress":
```

Comment:
maybe it should be a function in utils? we have this code in several places no?

Reply:
Done.